app/indexers/administrative_tag_indexer.rb

```diff
@@ -24,10 +24,15 @@ def to_solr
       solr_doc['exploded_tag_ssim'] += exploded_tags_from(tag)
 
       tag_prefix, rest = tag.split(TAG_PART_DELIMITER, 2)
-      next if TAGS_TO_INDEX.exclude?(tag_prefix) || rest.nil?
+      next if SPECIAL_TAG_TYPES_TO_INDEX.exclude?(tag_prefix) || rest.nil?
 
       prefix = tag_prefix.downcase.strip.gsub(/\s/, '_')
+
       (solr_doc["#{prefix}_tag_ssim"] ||= []) << rest.strip
```

Comment:
this is the line that was already doing the first part of what the ticket asked for